index.bs

```diff
@@ -839,6 +883,11 @@ To <dfn noexport>parse trigger-registration JSON</dfn> given a [=string=]
         <a spec="html">rules for parsing non-negative integers</a> to
         |value|["`debug_key`"].
     1. If |debugKey| is an error, set |debugKey| to null.
+1. Let |filters| be a new [=attribution filter map=].
```

Comment:
Unrelated:

Now that the headers are collapsed should we support top level negation on the trigger side? Seems like there is no reason not to.

Reply:
We could easily do so, but let's discuss in a separate bug.